lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Settings.hs

```diff
@@ -76,3 +82,40 @@ spec = describe "SHELLEY_SETTINGS" $ do
                     (`shouldBe` (either (const (error "no")) id $ fromText
                         @PoolMetadataSource uri))
                 ]
+
+    it "SETTINGS_02 - Changing pool_metadata_source re-syncs metadata" $ \ctx -> do
+        let toNone = "none"
+            toDirect = "direct"
+            getMetadata = map (view #metadata) . snd <$> unsafeRequest
```

Comment:
I personally prefer `fmap` over `map`, because even if we change from list to set or nonempty list, it still compiles... but ymmv

Reply:
done.